frontend/src/scenes/onboarding/OnboardingBillingStep.tsx

```diff
@@ -65,15 +65,18 @@ export const OnboardingBillingStep = ({
                     {product.subscribed && (
                         <div className="mb-8">
                             <div className="bg-success-highlight rounded p-6 flex justify-between items-center">
-                                <div className="flex gap-x-4">
-                                    <IconCheckCircle className="text-success text-3xl mb-6" />
-                                    <div>
+                                <div className="flex gap-x-4 min-w-0">
+                                    {/* min-w-0 to prevent overflow */}
```

Comment:
generally you don't need to leave comments for CSS changes, so I'd remove those.

If you do leave comments on logic to explain it, we generally put the comment above the line in question 👍

Reply:
removed the comments. Thanks for the feedback